mavros_extras/src/plugins/fake_gps.cpp

```diff
@@ -266,6 +225,9 @@ class FakeGPSPlugin : public plugin::PluginBase,
 		old_ecef = current_ecef;
 
 		UAS_FCU(m_uas)->send_message_ignore_drop(fix);
+
+		// Throttle incoming messages to 5hz
+		gps_rate.sleep();
```

Comment:
There are three ways of we can throttle the msg: one is the way it was before, the other is using `Ros::Timer` and the other is using `Ros::Rate`, which I'm doing now.
http://answers.ros.org/question/194418/change-publish-rate-of-a-topic/